test/Sentry.Maui.Tests/SentryMauiOptionsTests.cs

```diff
@@ -49,4 +49,11 @@ public void CacheDirectoryPath_Default()
         Assert.Equal(expected, options.CacheDirectoryPath);
 #endif
     }
+
+    [Fact]
+    public void AttachScreenshots_Default()
+    {
+        var options = new SentryMauiOptions();
+        Assert.False(options.AttachScreenshots);
```

Comment:
Some other tests that would help here:

1. Scope.clear doesn't affect screenshot capture
2. Setting it to true and an event is capture actually ends up with a screenshot as attachment (can verify on the transport level)
3. Remove screenshot from hint on beforeSend makes sure it doesn't show up on the transport later